src/app/onion/learning-object-builder/components/builder-navbar/builder-navbar.component.html

```diff
@@ -1,84 +1,181 @@
+      <clark-editor-action-panel
+        *ngIf="adminMode"
+        [learningObject]="learningObject"
+      ></clark-editor-action-panel>
     </div>
   </div>
   <div class="builder-navbar-wrapper__bottom">
     <div class="builder-navbar-wrapper__bottom-left-menu">
       <!-- update routerLinkActive with class used to show that link is active -->
       <ul>
         <li>
-          <ng-container [ngTemplateOutlet]="navItemTemplate" [ngTemplateOutletContext]="{ disabled: false, link: 'info', text: 'Basic Information' }"></ng-container>
+          <ng-container
+            [ngTemplateOutlet]="navItemTemplate"
+            [ngTemplateOutletContext]="{
+              disabled: false,
+              link: 'info',
+              text: 'Basic Information'
+            }"
+          ></ng-container>
         </li>
         <li>
-          <ng-container [ngTemplateOutlet]="navItemTemplate" [ngTemplateOutletContext]="{ disabled: !canRoute('outcomes'), link: 'outcomes', text: 'Learning Outcomes' }"></ng-container>
+          <ng-container
+            [ngTemplateOutlet]="navItemTemplate"
+            [ngTemplateOutletContext]="{
+              disabled: !canRoute('outcomes'),
+              link: 'outcomes',
+              text: 'Learning Outcomes'
+            }"
+          ></ng-container>
         </li>
         <li>
-          <ng-container [ngTemplateOutlet]="navItemTemplate" [ngTemplateOutletContext]="{ disabled: !canRoute('materials'), link: 'materials', text: 'Materials' }"></ng-container>
+          <ng-container
+            [ngTemplateOutlet]="navItemTemplate"
+            [ngTemplateOutletContext]="{
+              disabled: !canRoute('materials'),
+              link: 'materials',
+              text: 'Materials'
+            }"
+          ></ng-container>
         </li>
       </ul>
       <div class="builder-navbar-wrapper__saving-indicator">
-        <span *ngIf="isSaving" class="builder-navbar-wrapper__saving-indicator">Saving changes . . . </span>
-        <span *ngIf="!isSaving" class="builder-navbar-wrapper__saving-indicator">All changes saved</span>
+        <span *ngIf="isSaving" class="builder-navbar-wrapper__saving-indicator"
+          >Saving changes . . .
+        </span>
+        <span *ngIf="!isSaving" class="builder-navbar-wrapper__saving-indicator"
+          >All changes saved</span
+        >
       </div>
     </div>
-
   </div>
 </div>
 
-<clark-popup *ngIf="showSubmission" (closed)="showSubmission = false">
-  <div #popupInner>
-    <clark-collection-selector [currentCollection]="store.learningObjectEvent.getValue().collection" (choice)="submitForReview($event)"
-      (cancel)="showSubmission = false"></clark-collection-selector>
-  </div>
-</clark-popup>
+<clark-submit *ngIf="showSubmission" [learningObject]="store.learningObject" [visible]="true" (close)="showSubmission = false; getCollectionName()"></clark-submit>
```

Comment:
Most of the changes in this file are due to formatting. This is an actual change.

Reply:
Why do we need to get the collection name on close?